### ISSUE TITLE
Use CreateGroupingWorkspace for North and South banks instead of cal files in EngDiff UI

### ISSUE THREAD

**Issue by RichardWaiteSTFC** [mantidproject/mantid#31910]
Bit quicker as don't have to read file. 
A MWE is
```
Load(Filename=r'\\isis.cclrc.ac.uk\inst$\ndxenginx\instrument\data\cycle_12_4\ENGINX00193749.nxs', OutputWorkspace='193749')
ConvertUnits(InputWorkspace='193749', OutputWorkspace='193749', Target='dSpacing')
CreateGroupingWorkspace(InputWorkspace='193749', GroupNames='NorthBank', 
    OutputWorkspace='grouping')
DiffractionFocussing(InputWorkspace='193749', OutputWorkspace='foc', 
    GroupingWorkspace='grouping')
```

**Comment by RichardWaiteSTFC** [comment]
This might be addressed in #31778

**Comment by clayton-tom** [comment]
Fixed in #31778
